recipes/libmeshb/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, replace_in_file,  rmdir
+import os
+
+required_conan_version = ">=1.53.0"
+
+class LibmeshbConan(ConanFile):
+    name = "libmeshb"
+    description = "A library to handle the *.meshb file format."
+    license = "MPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/LoicMarechal/libMeshb"
+    topics = ("3d", "mesh", "geometry")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_gmf_asio": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_gmf_asio": False
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
```

Comment:
```suggestion
            del self.options.fPIC
            del self.options.with_gmf_asio
```

The option `gmf_asio` will require `aio.h` which is available only for Unix systems.

https://github.com/LoicMarechal/libMeshb/blob/v7.80/sources/libmeshb7.c#L114